docker-compose/scripts/run-dev.sh

```diff
@@ -46,10 +46,17 @@ fi
 
 echo "[$(date --rfc-3339 seconds)] - Starting Kuzzle..."
 
+if [ ! -z "$KUZZLE_PLUGINS" ];
```

Comment:
NP - Shellcheck suggestion:
```suggestion
if [ -n "$KUZZLE_PLUGINS" ];
```